### DISCUSSION TITLE
To frame or not to frame, that is the question...

### DISCUSSION THREAD

**Discussion by jingato** [awawa-dev/HyperHDR#698] [General]
Hello everyone,

I'm going to be building a new system and I'm currently parting it all out. One thing I am not sure about is the frame. My current system is an off the shelf Lytmi ( it sucks ) and I am replacing it with a HyperHDR Raspberry Pi system. So my current system is just attached to the back of the TV  via double sided tape with the leds pointing at the wall. I'm curious about a few things regarding a frame. For reference, my TV is a 65" OLED mounted to the wall and the screen is about 10 feet away from the viewer and 4 inches from the wall.

1) Is it really that much better to have a frame vs no frame?
2) What determines the angle to use? is 45 degrees usually the best?
3) Does the mount attach to the wall or the TV?
4) Does the angle point towards the wall or towards the viewer?
5) How much smaller should the frame be from the size of the TV?

Lastly, has anybody made a template that can be used to have one 3D printed or is wood the best solution?

Thanks!

John

**Comment by AstaRom** [comment]
> Lastly, has anybody made a template that can be used to have one 3D printed or is wood the best solution?

The wooden frame or plastic panel of the TV itself is far from the best material for dissipating heat.
The best solution would be an aluminum frame.
It should remove the heat generated by the LEDs during operation, which should extend the life of your LED strip.

**Comment by MoWerr** [comment]
What did you decide in the end? I'm facing similar questions right now and have no idea where to get the answers from.

Cheers!
